source/faq.txt

```diff
@@ -20,20 +20,18 @@ Support.
 Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
 ---------------------------------------------------------------------------------------
 
-You can perform reads during synchronization if ``canCommit`` is ``true``. 
-However, the data that you read is 
-:term:`eventually consistent <eventual consistency>`, meaning that you might not 
-always read the latest writes.
+You can perform reads during synchronization at any time. However, the data 
+that you read is :term:`eventually consistent <eventual consistency>`. For 
+up-to-date reads, wait for the migration to commit. To learn more, see 
```

Comment:
apologies that I missed this earlier, we should change "up-to-date" with "consistent" here as well, for the same reasons as the [prior thread](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/377#discussion_r1755426401) on the 'latest writes' verbiage that we're deleting.
no rush to update the staging on this new change BTW, I've shared the staging env with the team and I've mentioned this note above as well.